src/Processors/Formats/Impl/Parquet/Write.cpp

```diff
@@ -396,6 +398,45 @@ struct ConverterNumberAsFixedString
     size_t fixedStringSize() { return sizeof(T); }
 };
 
+struct ConverterJSON
+{
+    using Statistics = StatisticsStringRef;
+
+    const ColumnObject & column;
+    DataTypePtr data_type;
+    PODArray<parquet::ByteArray> buf;
+    std::vector<String> stash;
+
+    explicit ConverterJSON(const ColumnPtr & c, const DataTypePtr & data_type_)
+        : column(assert_cast<const ColumnObject &>(*c))
+        , data_type(data_type_)
+    {
+    }
+
+    const parquet::ByteArray * getBatch(size_t offset, size_t count)
+    {
+        buf.resize(count);
+        stash.clear();
+        stash.reserve(count);
+
+        auto internal_type = data_type ? data_type : std::make_shared<DataTypeObject>(DataTypeObject::SchemaFormat::JSON);
+        auto serialization = internal_type->getDefaultSerialization();
+        FormatSettings fmt;
```

Comment:
We can't use default format settings, let's propagate format settings from `ParquetBlockOutputFormat` through `writeColumnChunkBody`